docs/backup_(btrfs,crashplan).md

```diff
@@ -0,0 +1,79 @@
+# Backup w/ BTRFS & CrashPlan
+This document is intended to provide an example of how to setup a system to store large amounts of data with with flexibility, reasonably simple backup and recovery at a decent price.
+
+## Technologies
+* [GNU/Linux (Ubuntu 17.10.1)](https://ubuntu.com)
+* [BTRFS](https://btrfs.wiki.kernel.org)
+
+#### What and Why?
+There are many technologies in the storage space and the ease of use, cost, and reliability vary greatly. Here we'll attempt to articulate why these particular technologies were used versus others available.
+
+##### BTRFS
+---
+###### What?
+Btrfs is a modern copy on write (CoW) filesystem for Linux aimed at implementing advanced features while also focusing on fault tolerance, repair and easy administration.
+###### Why?
+BTRFS will be used for pooling and also for the filesystem. The data modell will be RAID5, but BTRFS has the flexbility to switch between data models if needed eg. RAID1 to RAID5/6.
+* **Why not EXT4/XFS?**
```

Comment:
Might want to expand this. The logical volume management, snapshotting, bitrot detection, etc.